Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -1411,4 +1411,19 @@ void VimEditingEngine::move_to_next_empty_lines_block()
     m_editor->set_cursor(new_cursor);
 };
 
+void VimEditingEngine::casefold_selection(Casing case_type)
```

Comment:
`Casing casing` or `CaseType case_type`. Pick one :)

Reply:
Casing casing it is. Thanks for merge :^)